buildspec-release.yml

```diff
@@ -1,111 +1,34 @@
 version: 0.2
 
-env:
-  variables:
-    FRAMEWORK_FULL_VERSION: '1.6.0'
-    FRAMEWORK_SHORT_VERSION: '1.6'
-    AWS_DEFAULT_REGION: 'us-west-2'
-    ECR_REPO: 'sagemaker-mxnet-serving'
-    GITHUB_REPO: 'sagemaker-mxnet-serving-container'
-    EI_ACCELERATOR_TYPE: 'ml.eia1.medium'
-    GPU_INSTANCE_TYPE: 'p2.xlarge'
-    SETUP_FILE: 'setup_cmds.sh'
-    SETUP_CMDS: '#!/bin/bash\npip install --upgrade pip\npip install -U -e .\npip install -U -e .[test]'
-
 phases:
-  pre_build:
-    commands:
-      - start-dockerd
-
-      - ACCOUNT=$(aws --region $AWS_DEFAULT_REGION sts --endpoint-url https://sts.$AWS_DEFAULT_REGION.amazonaws.com get-caller-identity --query 'Account' --output text)
-
   build:
     commands:
       # prepare the release (update versions, changelog etc.)
-      - git-release --prepare --min-version 1.0.1
+      - git-release --prepare
```

Comment:
pretty sure you're going to want the `--min-version` flag or else it'll start at like 0.1.0? (or maybe now that we've had automated "releases" already, the numbering will work fine)

Reply:
oh... that's what it's for!

Reply:
Next one should be something like v1.2.0